src/app/pcasts/dao/episodes_dao.py

```diff
@@ -95,3 +104,8 @@ def get_top_episodes_by_recommenders(offset, max_search, user_id):
   ]
   found_episodes = get_episodes(found_episode_ids, user_id)
   return order_by_ids(found_episode_ids, found_episodes)
+
+def get_series_ids_from_episodes(episode_ids):
```

Comment:
does this return the correct order?

Reply:
The ordering is irrelevant since it is just used to create the mapping. No ordering for ReadNewEpisodeNotification since it just stores if something was read